classes/plugins/ThemePlugin.inc.php

```diff
@@ -552,7 +552,7 @@ public function getOptionValues($contextId) {
 		foreach ($this->options as $optionName => $optionConfig) {
 			$value = isset($values[$optionName]) ? $values[$optionName] : null;
 			// Convert values stored in the db to the type of the default value
-			if (!is_null($optionConfig->default)) {
+			if ($value !== null && $optionConfig->default !== null) {
```

Comment:
The default value is used at the end of the `Field::getConfig()`, see:
https://github.com/pkp/pkp-lib/blob/d36f60ade0f546aeb570386deabb19ab4c282094/classes/components/forms/Field.inc.php#L125-L129

So, we shouldn't do coercions here, otherwise the field will have a non-null value and its default value will not be used later on.